www/_template/reference/configuration.md

```diff
@@ -124,211 +337,239 @@ module.exports = {
-- `resolve` _optional, default: true_: If false, don't resolve JS & CSS imports in your JS, CSS, and HTML files. Instead send every import to the browser, as written. We recommend that you don't disable this unless absolutely necessary, since it prevents Snowpack from handling your imports to things like packages, JSON files, CSS modules, and more. Leaving resolve as `true` has minimal impact on performance.
+Set the name of the used function to create JSX elements.
 
-```js
-// Example: Advanced "mount" usage
-// snowpack.config.json
-{
-  "mount": {
-    // Same behavior as the "src" example above:
-    "src": {url: "/dist"},
-    // Mount "public" to the root URL path ("/*") and serve files with zero transformations:
-    "public": {url: "/", static: true, resolve: false}
-  }
-}
-```
+### buildOptions.jsxFragment
 
-### config.alias
+**Type**: `string`
 
-`object` (package: package or path)
+**Default**: `React.Fragment` (or `Fragment` if Preact import is detected)
 
-> Note: In an older version of Snowpack, all mounted directories were also available as aliases by default. As of Snowpack 2.7, this is no longer the case and no aliases are defined by default.
+Set the name of the used function to create JSX fragments.
 
-The `alias` config option lets you define an import alias in your application. When aliasing a package, this allows you to import that package by another name in your application. This applies to imports inside of your dependencies as well, essentially replacing all references to the aliased package.
+## config.testOptions
 
-Aliasing a local directory (any path that starts with "./") creates a shortcut to import that file or directory. While we don't necessarily recommend this pattern, some projects do enjoy using these instead of relative paths:
+Configure your tests
 
-```diff
--import '../../../../../Button.js';
-+import '@app/Button.js';
-```
+TODO: is this enough? should we be more specific like "how Snowpack runs tests?"
+
+### testOptions.files
+
+**Type**: `string[]`
+
+**Default**: `["__tests__/**/*", "**/*.@(spec|test).*"]`
+
+Specifies what files Snowpack recognizes as tests.
+
+All matching test files are scanned for installable dependencies during development, but excluded from both scanning and building in your final build.
+
+TODO: I think this is using glob matching?
+## config.experiments
+
+**Type**: `object` (option name: value)
+
+This section is experimental and not yet finalized. May change across versions.
+
+[See the code for options](https://github.com/snowpackjs/snowpack/blob/main/snowpack/src/types/snowpack.ts#L235)
+
+## config.\_extensionMap
+
+TODO: I think this matches input extentions to output but I am not sure
```

Comment:
@FredKSchott not sure about this one

Reply:
not actually a config value / hidden from the user